jaeger/charts/jaeger/values.yaml

```diff
@@ -8,6 +8,8 @@ collector:
     version: 0.1.11
     pullPolicy: Always
   # resources:
+  # -- address of the jaeger backend to send traces to
```

Comment:
Maybe a blank line is required right after `#resources:` for the new comment to be picked up by helm-doc